### DISCUSSION TITLE
I created a language extention for VSCode

### DISCUSSION THREAD

**Discussion by MinekPo1** [Its-Jakey/SCPP#4] [Show and tell]
source is [here](https://github.com/MinekPo1/SCPP-vscode)

right now no icon since I didn't want to take the joy of creating a logo for a language you created from you :3

**Comment by MinekPo1** [comment]
You can now install it [here](https://marketplace.visualstudio.com/items?itemName=minekpo1.scpp-vscode)

**Reply by Its-Jakey** [reply]
This will be incredibly handy! Can I put this in SCPP README.md? I'll credit you.

**Reply by Its-Jakey** [reply]
Could you add block comments? the comments are C-style so start=/* and end=*/

**Reply by MinekPo1** [reply]
Oh they weren't in the spec so I commented out the part of the preset adding those. And yeah go ahead

**Reply by MinekPo1** [reply]
done

**Reply by Its-Jakey** [reply]
> done

Awesome, thanks!

**Comment by Its-Jakey** [comment]
> right now no icon since I didn't want to take the joy of creating a logo for a language you created from you :3

I'll start work on an icon, I'm thinking there will be the text "C++" and then the scratch cat either with his arms around it, or his arm over it
